### PR TITLE
#17: Use makeStyle instead of style (will update more once confirmed)

### PR DESCRIPTION
**Pull request by deepakchethan**
Fixes: https://github.com/meedan/check/issues/17

### CODE REVIEW

**Comment by deepakchethan** [comment]
@amoedoamorim, Can you please take a look at these changes.

File: src/app/components/source/UserPrivacy.js [hunk 1]
```diff
@@ -17,6 +18,30 @@ import { stringHelper } from '../../customHelpers';
...
+  },
+  headerStyle: {
+    margin: `${units(2)} 0`,
+    marginTop: units(6),
+  },
+  style: {
+    margin: `${units(2)} 0`,
+  },
+  cardStyle: {
+    margin: `${units(2)} 0`,
```
Comment (amoedoamorim) [inline src/app/components/source/UserPrivacy.js:33]: Great work, thank you! Can I ask you to please also remove these `units` call and the import in favor of the `theme.spacing` usage?

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Hi @deepakchethan ! Sorry for taking this long to review! Your PR looks amazing! Thanks so much for your contribution! Can I ask you to replace the `units` usage so it is 100% perfect? Thank you for participating!

File: src/app/components/source/UserPrivacy.js (hunk 1 above)
Comment (deepakchethan) [reply src/app/components/source/UserPrivacy.js:33]: Sure, I'll fix this.

File: src/app/components/source/UserPrivacy.js [hunk 2]
```diff
@@ -14,9 +16,32 @@ import CheckContext from '../../CheckContext';
 import { mapGlobalMessage } from '../MappedMessage';
 import { getErrorMessage } from '../../helpers';
 import { stringHelper } from '../../customHelpers';
-import { units } from '../../styles/js/shared';
 import globalStrings from '../../globalStrings';
 
+const useStyles = theme => ({
```
Comment (deepakchethan) [inline src/app/components/source/UserPrivacy.js:21]: Post update:

Screen Shot 2020-10-11 at 11 34 45 AM

File: src/app/components/FlashMessage.js [hunk 3]
```diff
@@ -61,8 +64,7 @@ const FlashMessage = () => {
     <Message
       message={message}
       onClick={resetMessage}
-      className="home__message"
-      style={flashMessageStyle}
+      className={`home__message ${classes.flashMessageStyle}`}
```
Comment (deepakchethan) [inline src/app/components/FlashMessage.js:67]: Post update:

Screen Shot 2020-10-11 at 12 00 17 PM

File: src/app/components/layout/ColorPicker.js [hunk 4]
```diff
@@ -7,25 +7,25 @@ import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
...
+    backgroundColor: props.backgroundColor,
+  }),
   statusButtonIcon: {
     fontSize: 32,
   },
 }));
 
 const ColorPicker = ({ color, onChange }) => {
-  const classes = useStyles();
+  const classes = useStyles({ backgroundColor: color });
```
Comment (deepakchethan) [inline src/app/components/layout/ColorPicker.js:23]: Post update:

Screen Shot 2020-10-11 at 12 07 24 PM

**Comment by deepakchethan** [comment]
Hello @amoedoamorim, I have made the requested changes as suggested and added corresponding updated screenshots for your reference.

**Comment by deepakchethan** [comment]
@amoedoamorim ping

File: src/app/components/source/UserPrivacy.js [hunk 5]
```diff
@@ -1,11 +1,13 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { compose } from 'redux';
```
Comment (amoedoamorim) [inline src/app/components/source/UserPrivacy.js:3]: Hello @deepakchethan! Sorry, I got a bit busy with some other stuff! Thanks for pinging!
Is this really necessary? We're trying to get rid of redux

Comment (amoedoamorim) [reply src/app/components/source/UserPrivacy.js:3]: Are you still around @deepakchethan?

Comment (deepakchethan) [reply src/app/components/source/UserPrivacy.js:3]: Hey @amoedoamorim, Sorry was not around. Will update the code

Comment (deepakchethan) [reply src/app/components/source/UserPrivacy.js]: Have removed it

**Comment by deepakchethan** [comment]
@amoedoamorim, I have removed redux. Please see if this pr is good for merging.

**Comment by amoedoamorim** [comment]
> @amoedoamorim, I have removed redux. Please see if this pr is good for merging.

Thanks! That looks good. Will run tests on it and if it all passes, I'll merge it!

**Comment by amoedoamorim** [comment]
Congrats on your first accepted PR @deepakchethan! We appreciate your contribution! :smile::tada:

**Pull request merged**